test/server/cards/05_LOF/upgrades/InquisitorsLightsaber.spec.ts

```diff
@@ -0,0 +1,34 @@
+describe('Inquisitor\'s Lightsaber', function () {
+    integration(function (contextRef) {
+        it('Inquisitor\'s Lightsaber\'s should give +2/+0 while attacking a force unit', async function () {
+            await contextRef.setupTestAsync({
+                phase: 'action',
+                player1: {
+                    groundArena: [{ card: 'yoda#old-master', upgrades: ['inquisitors-lightsaber'] }],
+                },
+                player2: {
+                    groundArena: ['rey#keeping-the-past', 'consular-security-force'],
+                }
+            });
+
+            const { context } = contextRef;
+
+            context.player1.clickCard(context.yoda);
+            context.player1.clickCard(context.consularSecurityForce);
+
+            expect(context.player2).toBeActivePlayer();
+            expect(context.consularSecurityForce.damage).toBe(3); // 2+1
+
+            context.setDamage(context.yoda, 0);
+            context.yoda.exhausted = false;
```

Comment:
This needs to use the context to ensure the state is refreshed correctly:
```suggestion
            context.readyCard(context.yoda);
```